ecommerce/frontend/src/components/pages/LoginPage.js

```diff
@@ -1,3 +1,107 @@
-export const LoginPage = () => {
-    return 'login'
-}
+import './LoginPage.less'
+import { Form, Input, Button, Row, Col } from 'antd'
+import { Link } from 'react-router-dom'
+import Column from 'antd/lib/table/Column'
+
+export const LoginPage = ({ onSubmit = () => { } }) => {
```

Comment:
not necessarily a problem but this can be named as LoginForm rather than LoginPage,.

Reply:
Agreeing with Ozdeniz. Building a separate component LoginForm, then integrating it inside the Login Page is a better practice. 
```
export const LoginPage = () => {
    return <LoginForm />
}
```

Reply:
Agree too.